.circleci/config.yml

```diff
@@ -41,7 +41,22 @@ jobs:
       - run:
           command: echo "starting build"
       - save_root_build_number
-
+  commit-lint-branch:
+    executor: default_executor
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "ee:b4:00:e6:23:5b:55:bb:fd:07:bc:73:9e:f7:89:9c" # shared.aspire@talis.com 'talis-cdk-constructs Deploy Key'
+      - checkout
+      - node/install-packages
+      - run:
+          name: Lint all commits on the branch
+          command: |
+            FIRST_COMMIT=$(git merge-base main $CIRCLE_BRANCH)
+            LAST_COMMIT=$(git rev-parse HEAD)
```

Comment:
Is that how we found the commit range previously?

Reply:
no, AFAIK we were never checking a range, only the last commit.